app/scripts/templates/search/index.html

```diff
@@ -0,0 +1,11 @@
+<h3 class="number-of-results">
+  {{#onlyOneResult}}
+    {{numberOfResults}} result
+  {{/onlyOneResult}}
+  {{^onlyOneResult}}
+    {{numberOfResults}} results
+  {{/onlyOneResult}}
```

Comment:
Probably not the best place for my kvetching, but we need a better state for "0 results" as it looks pretty barren.

Or tell me to shut up and I'll file a new bug and we can unleash the UX wonder-twins on it.